### ISSUE TITLE
What is the recommended way to register the name of a person?

### ISSUE THREAD

**Issue by danse** [eHealthAfrica/eHealthAfrica.github.io#19]
Considering for example input fields on a form, should they be:

1- `name`
2- `name`, `surname`
3- `name`, `other names`
4.- ... ?

**Comment by cristinasantamarina** [comment]
4 - first name, last name (NutSurv)
5 - name, surname, prename (HAT)

**Comment by danse** [comment]
@cristinasantamarina hum, these are different alternatives. The point is not how we are inconsistently doing this now, but what is the recommended way of doing this in the future. For example, using just a single `name` field in the future could save us many problems, supporting easily cultures with arbitrarily complex naming conventions

**Comment by danse** [comment]
oh you just wanted to extend the collection ... allright sorry :)

**Comment by cristinasantamarina** [comment]
@danse Yes just pointing out which other alternatives we use. I think if we want to set an standard it is worth looking at what we have. 

Is the idea to replace the current models or to aim at using a standard from now on?

**Comment by danse** [comment]
Just a recommendation for the future, answering to questions like the recent one from @lucychambers 

> What is conventional wisdom on how to do name fields? One field for ‘name’ or two ‘First name / Last name’
